sympy/concrete/summations.py

```diff
@@ -174,6 +174,21 @@ def _eval_is_zero(self):
         if self.function.is_zero:
             return True
 
+    def _eval_is_extended_real(self):
+        return self.function.is_extended_real
+
+    def _eval_is_positive(self):
+        return self.function.is_positive
```

Comment:
Similar comments about infinite sums. Also we have to consider the empty sum here. I just went to show an example and got these:
```julia
In [8]: Sum(1, (x, 0, 0)).doit()                                                                                                  
Out[8]: 1

In [9]: Sum(1, (x, 0, -1)).doit()                                                                                                 
Out[9]: 0

In [10]: Sum(1, (x, 0, -5)).doit()                                                                                                
Out[10]: -4
```
I don't know if I think those are correct though...

Reply:
Inverted limits are clearly worse... I was under the impression that these were inverted, but I see it is in doit, not on initialization.

Reply:
Should still be some cases where it is possible to determine that it works.

Reply:
I still understand the convention for Sum limits well enough to answer this: how do I define an empty Sum?

Reply:
I do not think that it is feasible, unless it is the same as having a negative difference of one.